### PR TITLE
Added nooponer noreferrer for preacaution against reverse tabnagging

### LINKED ISSUE #214: Include noreferrer and noopener to all external links
Include target _blank, noreferrer and noopener to all external links, for example in :
https://github.com/ossn/ossn-frontend/blob/92c0fd9307b1833d7e7e9364976baf119b75f66e/src/components/navigation/footer-navigation/footer-navigation.js#L52

### PR DESCRIPTION
**Pull request by shakti97**
It closes #214 issue

### CODE REVIEW

File: src/components/layouts/auth-wrapper/auth-wrapper.js [hunk 1]
```diff
@@ -32,6 +32,8 @@ export const LoginLink = ({ label = "Login", className = "" }) => (
   <a
     className={className}
     href={`${BACKEND_URL}/oidc/login`}
+    target="_blank"
+    rel="noopener noreferrer"
```
Comment (alexdor) [inline src/components/layouts/auth-wrapper/auth-wrapper.js]: this link is for the login so we shouldn't include them here

File: src/components/components/join-cta/join-cta.js [hunk 2]
```diff
@@ -20,7 +20,12 @@ const JoinCta = props => {
     props.className
   }`;
   return (
-    <a href={`${BACKEND_URL}/oidc/login`} className={classString}>
+    <a
+      href={`${BACKEND_URL}/oidc/login`}
+      target="_blank"
+      rel="noopener noreferrer"
+      className={classString}
```
Comment (alexdor) [inline src/components/components/join-cta/join-cta.js]: this link is for the login so we shouldn't include them here

File: src/components/components/leader-tools/leader-tools.js [hunk 3]
```diff
@@ -33,7 +33,13 @@ const LeaderTool = props => {
 
   return (
     <li className="leader-tool">
-      <a href={url} className="leader-tool__link" title={title}>
+      <a
```
Comment (alexdor) [inline src/components/components/leader-tools/leader-tools.js]: I don't remember if this is an external link or not, could you verify that this is an external link?

File: src/components/components/member-training-resources/member-training-resources.js [hunk 4]
```diff
@@ -29,7 +29,12 @@ const TrainingResource = props => {
 
   return (
     <div className={classes}>
-      <a href={url} className="member-training-resources__wrapper-link">
```
Comment (alexdor) [inline src/components/components/member-training-resources/member-training-resources.js]: I don't remember if this is an external link or not, could you verify that this is an external link?

File: src/components/components/teaser-box/teaser-box.js [hunk 5]
```diff
@@ -30,7 +30,12 @@ export const TeaserBox = props => {
 
   return (
     <div className={classes.join(" ")}>
-      <a href={target} className="teaser-box__wrapper-link">
```
Comment (alexdor) [inline src/components/components/teaser-box/teaser-box.js]: I don't remember if this is an external link or not, could you verify that this is an external link?

File: src/components/layouts/auth-wrapper/auth-wrapper.js (hunk 1 above)
Comment (shakti97) [reply src/components/layouts/auth-wrapper/auth-wrapper.js]: okay

File: src/components/components/join-cta/join-cta.js (hunk 2 above)
Comment (shakti97) [reply src/components/components/join-cta/join-cta.js]: okay

File: src/components/components/leader-tools/leader-tools.js (hunk 3 above)
Comment (shakti97) [reply src/components/components/leader-tools/leader-tools.js]: yes sure

File: src/components/components/member-training-resources/member-training-resources.js (hunk 4 above)
Comment (shakti97) [reply src/components/components/member-training-resources/member-training-resources.js]: yes sure

File: src/components/components/teaser-box/teaser-box.js (hunk 5 above)
Comment (shakti97) [reply src/components/components/teaser-box/teaser-box.js]: yes sure

File: src/components/components/leader-tools/leader-tools.js (hunk 3 above)
Comment (shakti97) [reply src/components/components/leader-tools/leader-tools.js]: this is an external link so we need that here

File: src/components/components/member-training-resources/member-training-resources.js (hunk 4 above)
Comment (shakti97) [reply src/components/components/member-training-resources/member-training-resources.js]: Its an external link, so we need that here

File: src/components/components/teaser-box/teaser-box.js (hunk 5 above)
Comment (shakti97) [reply src/components/components/teaser-box/teaser-box.js]: This is not an external link so i removed that from here

**Comment by shakti97** [comment]
Done with the recommended changes

**Pull request merged**